test_util/src/lib.rs

```diff
@@ -1929,7 +1929,7 @@ pub struct CheckOutputIntegrationTest<'a> {
   /// the test creates files in the testdata directory (ex. a node_modules folder)
   pub copy_temp_dir: Option<&'a str>,
   /// Relative to "testdata" directory
-  pub maybe_cwd: Option<&'a str>,
+  pub cwd: Option<&'a str>,
```

Comment:
Ok 🤷

Reply:
This property is optionally provided and never consumed or non-optionally provided by the caller, so I found it weird to say `maybe`. I think `maybe_` makes sense in other places though.